checkmate/templates/admin/base.html.jinja2

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Checkmate Admin</title>
+  <style>
```

Comment:
I reckon we should use bulma here like the LMS admin pages:

https://bulma.io/

I don't think we need to go overboard with styling, just slap the CSS import and tweak the HTML a bit to render it a bit nicely


See the base file in LMS:  https://github.com/hypothesis/lms/blob/main/lms/templates/admin/base.html.jinja2#L34


I'd do in a follow up PR.

Reply:
Sounds good no problem!